build-rpi-raspbian/webif.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+# start:     	starts a minimalistic web server
+# 	    	shows the status of OpenHABianPi installation
+# restart:   	checks if webserver is running 
+# inst_done: 	create finish message and link to http://$hostname:8080
+# cleanup:	stops the webserver
+#	     	removes all no longer needed files
+
+hostname=openhabianpi
```

Comment:
doesn't have to be true, please take from openhabian.conf

Reply:
done